src/lib/constants/carousel-config.ts

```diff
@@ -0,0 +1,39 @@
+import { Settings } from 'react-slick';
+
+export const settings: Settings = {
+  dots: false,
+  arrows: false,
+  slidesToShow: 3,
+  slidesToScroll: 2,
+  responsive: [
+    {
+      breakpoint: 1200,
+      settings: {
+        slidesToShow: 2,
+        slidesToScroll: 1,
+      },
+    },
+    {
+      breakpoint: 800,
+      settings: {
+        slidesToShow: 1,
+        slidesToScroll: 1,
+        initialSlide: 2,
+      },
+    },
+  ],
+};
+
+interface Styles {
```

Comment:
interfaces should be names with `I` on the start.
```suggestion
interface IStyles {
```

it'd be nice to have eslint for this